src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -133,25 +141,35 @@ def export_config(cmd,
                   key=None,
                   prefix="",  # prefix to remove
                   yes=False,
+                  skip_features=False,
                   # to-file parameters
                   path=None,
                   format_=None,
                   separator=None,
+                  naming_convention='pascal',
                   # to-config-store parameters
                   dest_name=None,
                   dest_connection_string=None,
                   dest_label=None,
+                  preserve_labels=False,
                   # to-app-service parameters
-                  appservice_account=None,
-                  skip_features=False,
-                  naming_convention='pascal'):
+                  appservice_account=None):
     src_features = []
     dest_features = []
     dest_kvs = []
     destination = destination.lower()
     format_ = format_.lower() if format_ else None
     naming_convention = naming_convention.lower()
 
+    if destination == 'appconfig':
+        if dest_label is not None and preserve_labels:
+            raise CLIError("Export failed! Please provide only one of these arguments: 'dest-label' or 'preserve-labels'. See 'az appconfig kv export -h' for examples.")
```

Comment:
can you use '--dest-label' or '--preserver-labels' for error message? In addition, can we make this check part as validator and reuse the function?

Reply:
Sure, I can change the error message. As for the validator, we can't have a common validator because the argument names are different between import and export. 

---
In reply to: [365275540](https://github.com/Azure/azure-cli/pull/11756#discussion_r365275540) [](ancestors = 365275540)